scripts/cibuild

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${MAML_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Execute tests and build artifacts.
```

Comment:
We're not building artifacts here yet, right?

Reply:
No, not yet. I wrote the usage instructions as a placeholder for what's to come. I can fix that.

```suggestion
Execute tests.
```